crates/ubrn_cli/src/codegen/templates/ModuleTemplate.java

```diff
@@ -0,0 +1,54 @@
+{%- let android = self.config.project.android.clone() %}
+{%- let name = self.config.project.name_upper_camel() %}
+{%- let module_class_name = name|fmt("{}Module") %}
+// Generated by uniffi-bindgen-react-native
+package {{ android.package_name }};
+
+import androidx.annotation.NonNull;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.module.annotations.ReactModule;
+
+@ReactModule(name = {{ module_class_name }}.NAME)
+public class {{ module_class_name }} extends {{ self.config.project.tm.spec_name() }} {
+  public static final String NAME = "{{ name }}";
+
+  public MatrixRustSdkModule(ReactApplicationContext reactContext) {
+    super(reactContext);
+  }
+
+  @Override
+  @NonNull
+  public String getName() {
+    return NAME;
+  }
+
+  static {
+    System.loadLibrary("{{ self.config.project.cpp_filename() }}");
+  }
+
+  private static native double nativeMultiply(double a, double b);
+  private static native boolean nativeInstallRustCrate(long rtPtr, boolean a);
+  private static native boolean nativeCleanupRustCrate(long rtPtr, boolean a);
+
+  @Override
```

Comment:
This is a placeholder right, eventually it gets deleted (maybe a TODO note is warranted)

Reply:
The `nativeInstallRustCrate` and `nativeCleanupRustCrate` is constant. The `multiply` is temporary, however.